ast/unitnormmap.h

```diff
@@ -0,0 +1,305 @@
+*     - apply the same UnitNormMap in the inverse direction to produce the result
+*     
+
+*  Inheritance:
+*     The UnitNormMap class inherits from the Mapping class.
+
+*  Attributes Over-Ridden:
+*     None.
+
+*  New Attributes Defined:
+*     None.
+
+*  Methods Over-Ridden:
+*     Public:
+*        None.
+*
+*     Protected:
+*        ClearAttrib
+*           Clear an attribute value for a UnitNormMap.
+*        GetAttrib
+*           Get an attribute value for a UnitNormMap.
+*        SetAttrib
+*           Set an attribute value for a UnitNormMap.
+*        TestAttrib
+*           Test if an attribute value has been set for a UnitNormMap.
+*        astMapMerge
+*           Simplify a sequence of Mappings containing a UnitNormMap.
+*        astTransform
+*           Apply a UnitNormMap to transform a set of points.
+
+*  New Methods Defined:
+*     Public:
+*        None.
+*
+*     Protected:
+*        None.
+
+*  Other Class Functions:
+*     Public:
+*        astIsAUnitNormMap
+*           Test class membership.
+*        astUnitNormMap
+*           Create a UnitNormMap.
+*
+*     Protected:
+*        astCheckUnitNormMap
+*           Validate class membership.
+*        astInitUnitNormMap
+*           Initialise a UnitNormMap.
+*        astInitUnitNormMapVtab
+*           Initialise the virtual function table for the UnitNormMap class.
+*        astLoadUnitNormMap
+*           Load a UnitNormMap.
+
+*  Macros:
+*     None.
+
+*  Type Definitions:
+*     Public:
+*        AstUnitNormMap
+*           UnitNormMap object type.
+*
+*     Protected:
+*        AstUnitNormMapVtab
+*           UnitNormMap virtual function table type.
+
+*  Feature Test Macros:
+*     astCLASS
+*        If the astCLASS macro is undefined, only public symbols are
+*        made available, otherwise protected symbols (for use in other
+*        class implementations) are defined. This macro also affects
+*        the reporting of error context information, which is only
+*        provided for external calls to the AST library.
+
+*  Copyright:
+*     Copyright (C) 1997-2006 Council for the Central Laboratory of the
+*     Research Councils
+
+*  Licence:
+*     This program is free software: you can redistribute it and/or
+*     modify it under the terms of the GNU Lesser General Public
+*     License as published by the Free Software Foundation, either
+*     version 3 of the License, or (at your option) any later
+*     version.
+*     
+*     This program is distributed in the hope that it will be useful,
+*     but WITHOUT ANY WARRANTY; without even the implied warranty of
+*     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*     GNU Lesser General Public License for more details.
+*     
+*     You should have received a copy of the GNU Lesser General
+*     License along with this program.  If not, see
+*     <http://www.gnu.org/licenses/>.
+
+*  Authors:
+*     DSB: D.S. Berry (Starlink)
```

Comment:
Good catch. I will update this and the .c file. I will also check the other files I had to modify for history info that needs updating.